docs/root/configuration/http/http_filters/header_to_metadata_filter.rst

```diff
@@ -72,8 +72,9 @@ Note that this filter also supports per route configuration:
       routes:
       - match: { prefix: "/version-to-metadata" }
         route: { cluster: service }
-        per_filter_config:
+        typed_per_filter_config:
           envoy.filters.http.header_to_metadata:
+            "@type": type.googleapis.com/envoy.extensions.filters.http.header_to_metadata.v3.Config
```

Comment:
Is this one a typo? The previous filter name is still here above the type..

Reply:
```envoy.filters.http.header_to_metadata``` you mean? ```typed_per_filter_config``` is a map, where the filter name is a key.